homeassistant/components/clementine/media_player.py

```diff
@@ -58,9 +58,9 @@
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Clementine platform."""
 
-    host = config.get(CONF_HOST)
-    port = config.get(CONF_PORT)
-    token = config.get(CONF_ACCESS_TOKEN)
+    host = config[CONF_HOST]
+    port = config[CONF_PORT]
+    token = config[CONF_ACCESS_TOKEN]
```

Comment:
`CONF_ACCESS_TOKEN` is optional and has no default.

Reply:
Good catch!